dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/dxf2/events/EventImportTest.java

```diff
@@ -166,12 +166,6 @@ class EventImportTest extends TransactionalIntegrationTest
 
     private static final SimpleDateFormat simpleDateFormat = new SimpleDateFormat( DateUtils.ISO8601_NO_TZ_PATTERN );
 
-    @Override
-    public boolean emptyDatabaseAfterTest()
```

Comment:
Is this unused? Or not needed anymore?
Cleaning up after tests seems like something we want 😬

Reply:
Sure, this is why empty the database is the default behaviour.
I am not even sure we should allow to override it.

Reply:
I don't think we should allow overriding it 🤔